server/src/core/user/index.ts

```diff
@@ -1,2 +1,4 @@
 export { User, UserCreateArgs, UserService, UserRole } from './types';
-export { Users } from './user-service';
\ No newline at end of file
+export { Users } from './user-service';
+export * as Validator from './validator';
+export * as Schema from './schema';
```

Comment:
The validation schemas should not be exported in `index.ts` because they're not used outside the user models. These are mere implementation-details of the validator. They could even live in the same file as the validator, but it's also okay to keep them in separate file. If you keep the files separate, then rename `schema` to `validation-schemas` because those schemas are an implementation detail of the validators.